### ISSUE TITLE
Ability to stop task

### ISSUE THREAD

**Issue by iamelevich** [go-task/vscode-task#165]
I want to have ability to manually stop the task. And see the list of currently running tasks.

For example I can stop long running task in terminal by Ctrl + C, but did not found a way to stop task started by VSCode extension.

Right now I need to find and kill that process, that only way that I found to stop the task.

**Comment by kosteklvp** [comment]
+1, this applies to running tasks with    `"outputTo": "output"`

**Comment by WAY29** [comment]
good idea, Is there anyone can implement this feature? i can only use config `"outputTo": "terminal"` for stop tasks
